src/test/streaming_test.cpp

```diff
@@ -449,6 +449,256 @@ node {
     ASSERT_EQ(executor.inferStream(this->firstRequest, this->stream), StatusCode::OK);
 }
 
+// Generative AI case + automatic timestamping server-side - Python
+
+#include <pybind11/embed.h>  // everything needed for embedding
+namespace py = pybind11;
+
+#include "../mediapipe_internal/pythonnoderesource.hpp"
+#include "../python/python_backend.hpp"
+
+static std::unordered_map<std::string, std::shared_ptr<PythonNodeResource>> prepareInputSidePacket(const std::string& handlerPath, PythonBackend* pythonBackend) {
```

Comment:
Are you sure we want to include OVMS source code in the test? What if OVMS code changes in the future? We will have outdated unit test.

Reply:
Please rebase, isnt't it better to create full stack test with model manager, MP factory, definition creation and KFS request?

Reply:
I'll do that